RLTest/redis_std.py

```diff
@@ -171,7 +168,7 @@ def _getRedisVersion(self):
 
     def createCmdArgs(self, role):
         cmdArgs = []
-        if self.debugger:
+        if self.debugger and role == MASTER and self.masterServerId == 1:
```

Comment:
I believe this breaks our valgrind run, now on cluster only the first shard runs with valgrind :\